### PR TITLE
Feat/implement pipingapp

### PR DESCRIPTION
**Pull request by MagnusGjosund**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@ dist/
 bin/
 # dependencies
 node_modules
+**/responsePiping.json
```
Comment (Gustav-Eikaas) [inline .gitignore:15]: Remove this?

File: apps/piping/app.config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const { name } = JSON.parse(readFileSync('./package.json').toString('utf-8'));
+
+export default () => ({
+  manifest: {
+    key: name,
+    name: name,
+  },
+  environment: {
+    // uri: 'https://backend-fusion-data-gateway-test.radix.equinor.com',
```
Comment (Gustav-Eikaas) [inline apps/piping/app.config.js:11]: Create an app.local.config.js and fix this one

File: apps/piping/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "piping",
+  "displayName": "Piping",
+  "version": "0.0.1",
+  "main": "/src/main.tsx",
+  "private": true,
+  "type": "module",
+  "scripts": {
```
Comment (Gustav-Eikaas) [inline apps/piping/package.json:8]: Look at another apps packagejson and update scripts. These are outdated

File: apps/piping/vite.config.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import viteConfigBase from '../../vite.config.base';
+export default {
+  ...viteConfigBase,
```
Comment (Gustav-Eikaas) [inline apps/piping/vite.config.ts:3]: Pretty sure viteconfig is outdated aswell

File: libs/pipingapp/src/lib/utils-table/generateCommaSeperatedStringArrayColumn.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+export function generateCommaSeperatedStringArrayColumn(
+  stringArray: string[],
+  maxLength?: number
+): string {
+  if (stringArray.length === 0) {
+    return '';
+  }
+  const length = maxLength ?? 3;
+  let commaString = stringArray.slice(0, length).join(', ');
```
Comment (Gustav-Eikaas) [inline libs/pipingapp/src/lib/utils-table/generateCommaSeperatedStringArrayColumn.ts:9]: Are you sure these files are being used?

**Review by Gustav-Eikaas — CHANGES_REQUESTED** [review]

File: libs/pipingapp/src/lib/utils-table/generateCommaSeperatedStringArrayColumn.ts (hunk 5 above)
Comment (MagnusGjosund) [reply libs/pipingapp/src/lib/utils-table/generateCommaSeperatedStringArrayColumn.ts:9]: not yet

**Review by Gustav-Eikaas — APPROVED** [review]

**Pull request merged**